demo/blocks_form/run.py

```diff
@@ -0,0 +1,32 @@
+import gradio as gr
+
+with gr.Blocks() as demo:
+    error_box = gr.Textbox(label="Error", visible=False)
+
+    name_box = gr.Textbox(label="Name")
+    age_box = gr.Number(label="Age")
+    symptoms_box = gr.CheckboxGroup(["Cough", "Fever", "Runny Nose"])
+    submit_btn = gr.Button("Submit")
+
+    diagnosis_box = gr.Textbox(label="Diagnosis")
+    patient_summary_box = gr.Textbox(label="Patient Summary")
+
+    def submit(name, age, symptoms):
+        if len(name) == 0:
+            return {error_box: gr.update(value="Enter name", visible=True)}
+        if age < 0 or age > 200:
+            return {error_box: gr.update(value="Enter valid age", visible=True)}
+        return {
+            diagnosis_box: "covid" if "Cough" in symptoms else "flu",
+            patient_summary_box: f"{name}, {age} y/o",
+        }
+
+    submit_btn.click(
+        submit,
+        [name_box, age_box, symptoms_box],
+        [error_box, diagnosis_box, patient_summary_box],
+    )
```

Comment:
I like the new way of update a lot, it is very intuitive and clear! Good job @aliabid94!

But I still have some concerns about we `gr.update`, it does the job but it does not link to the specific component, and it does not have any function signature that users could follow.

WDYT of these? I prefer 1.

1. Instead of gr.update, let's have `gr.{Component}.update`, ie. `gr.Textbox.update(value="Enter name", visible=True)`
2. (Or, have `input.update((value="Enter name", visible=True)`, in this case it would be like this:
   ```
def submit(name, age, symptoms, error_box):
        if len(name) == 0:
            return {error_box: error_box.update(value="Enter name", visible=True)}
        if age < 0 or age > 200:
            return {error_box: error_box.update(value="Enter valid age", visible=True)}
        return {
            diagnosis_box: "covid" if "Cough" in symptoms else "flu",
            patient_summary_box: f"{name}, {age} y/o",
        }

    submit_btn.click(
        submit,
        [name_box, age_box, symptoms_box, error_box],
        [error_box, diagnosis_box, patient_summary_box],
    )
```

Reply:
> Instead of gr.update, let's have gr.{Component}.update, ie. gr.Textbox.update(value="Enter name", visible=True)

We actually do have this @FarukOzderim! `gr.update()` is a catch all, but every component defines its own `update()` which includes type hints. 

@aliabid94 let's use `Textbox.update()`, etc. in our examples

Reply:
Great!

Reply:
Let's update this [blocks_update](https://github.com/gradio-app/gradio/blob/main/demo/blocks_update/run.py)as well

Reply:
@aliabid94 reminder in case you missed these messages.